src/include/stir/recon_buildblock/BackProjectorByBin.h

```diff
@@ -77,6 +77,13 @@ class BackProjectorByBin :
     const shared_ptr<DiscretisedDensity<3,float> >& density_info_sptr // TODO should be Info only
     ) =0;
 
+  /// Setup whilst applying a post-back-projection data processor
+  void set_up(
+     const shared_ptr<ProjDataInfo>& proj_data_info_ptr,
+     const shared_ptr<DiscretisedDensity<3,float> >& density_info_sptr, // TODO should be Info only
+     shared_ptr<DataProcessor<DiscretisedDensity<3,float> > > post_data_processor_sptr
+     );
```

Comment:
why do we have this? It's not going to be called anywhere. Seems wrong.

Instead, we should have a `set_post_data_processor_sptr()`.